components/ui/data-table/data-table.tsx

```diff
@@ -115,35 +115,35 @@ export function DataTable<TData, TValue>(props: DataTableProps<TData, TValue>) {
 
   return (
     <div>
-      <div className="flex items-center justify-between py-4">
-        <h2>
-          <b>Explore arbovirus seroprevalence estimates in our database</b>
-        </h2>
+      <div className="flex items-center justify-between py-4 px-2">
```

Comment:
Padding shows that table is missing a left border potentially nabd though. Is removal of top border intentional?

![image](https://github.com/serotracker/dashboard/assets/86806388/071ff2af-c5fc-44f0-81b1-e37b9139b8fd)